Jenkinsfile

```diff
@@ -1,50 +1,202 @@
-// this will need to be pipeline-library@master when the pr merges
-@Library('pipeline-library@ui-pipeline-legacy') _
+import groovy.json.JsonBuilder
+
+@Library('pipeline-library@COMUI-857') _
+
+def MAIN_BRANCH = 'master'
+def DEVELOP_BRANCH = 'develop'
```

Comment:
In order for this to work, you need to do a few things: 
1. Create a branch off `master` named `develop` and push it to origin
2. Submit a [ticket](https://inindca.atlassian.net/servicedesk/customer/portal/55) to have the new `develop` branch be the **default branch** for this repo (I can help you with this over chat)